From 679520443973f9fdb104f7f3f761e7dcf6ba7339 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Mon, 23 Dec 2019 14:31:06 +0200
Subject: [PATCH] [SW-6739] arcboot: remove cflags for sb_lib build

remove cflags for secureboot from arcboot_cflags as they dont effect
secureboot library from synopsis

Change-Id: I1e566943b660b322266daca4355747cc290ff3b3
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/Makefile b/Makefile
index 905353b..3c79a40 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@
 OBJCOPY = $(CROSS_COMPILE)objcopy
 
 LIBDIR = sdk/lib
-ARCBOOT_CFLAGS = -DSB_RSA_STD -DSB_FAST_MODE -DELP_SOURCE -Wall -W -Werror -O3
+ARCBOOT_CFLAGS = -Wall -W -Werror -O3
 ARCBOOT_CFLAGS += -Iinclude
 ARCBOOT_CFLAGS += -I$(HL_DEV)
 ARCBOOT_CFLAGS += -Isdk/src
